tests/aws_test.py

```diff
@@ -129,6 +150,67 @@ def testInstanceDeleted(self):
     util.IssueRetryableCommand.side_effect = [(json.dumps(response), None)]
     self.assertFalse(self.vm._Exists())
 
+  def testCreateSpot(self):
+    vm_util.IssueCommand.side_effect = [(self.sir_response, None, None),
+                                        (self.sir_response, None, None)]
+
+    self.vm._CreateSpot()
+
+    vm_util.IssueCommand.assert_any_call(
+        ['aws',
+         '--output',
+         'json',
+         'ec2',
+         'request-spot-instances',
+         '--region=us-east-1',
+         '--spot-price=123.45',
+         '--client-token=00000000-1111-2222-3333-444444444444',
+         '--launch-specification='
+         '{"ImageId":"ami-12345","InstanceType":"c3.large",'
```

Comment:
Nevermind about this -- you'll also have to split the text on newline and with all that it begins to look uglier than the original.